README.md

```diff
@@ -1 +1,128 @@
-# World-Chef-Capstone-project
\ No newline at end of file
+# World-Chef-Capstone-project
+
+<a name="readme-top"></a>
+
+# 📗Table of Contents
+
+- [📖 About the Project](#about-project)
+- [🛠 Built With](#built-with)
+- [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [👥 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [World Chef Conference Capstone Project] <a name="about-project"></a>
+
+**[World Chef Conference]** is a website which showcases content and information about the upcoming World Chef summit. It gives aspiring attendants the ability to register for the event upfront, get more detailed information about the event, get in touch with planners of the event.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- A Javascript file
+- An md file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+>
+
+- [Live Demo Link](https://github.com/Rachelwebdev/World-Chef-Capstone-project)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+A Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git@github.com:Rachelwebdev/Portfolio-Project.git`
+
```

Comment:
- I think it will be better to update your Readme's Setup. This still speaks about the portfolio. 🤗

Reply:
Thanks. I would update this now.